CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# dbt_shopify v0.12.1
+
+[PR #81](https://github.com/fivetran/dbt_shopify/pull/81) includes the following updates: 
```

Comment:
The latest PR is #82, so I'd change this from #81 to #82 or do what you did in the source package (call out the specific change that was applied in #81 on the bug fixes line).

Reply:
removed this line and added the PRs to the exact change line items